src/pages/patientView/mutation/column/TumorColumnFormatter.tsx

```diff
@@ -67,6 +91,23 @@ export default class TumorColumnFormatter {
         }, {} as {[s:string]:boolean});
     }
 
+    public static getProfiledSamplesForGene(entrezGeneId:number, sampleIds:string[], sampleToGenePanelId:{[sampleId: string]: string|undefined}, genePanelIdToEntrezGeneIds:{[genePanelId: string]: number[]}) {
+        // For a given gene indicate whether it was profiled in a particular sample
+        return sampleIds.reduce((map, nextSampleId, currentIndex:number) => {
+            const genePanelId = sampleToGenePanelId[nextSampleId];
+            
+            const wholeGenome = noGenePanelUsed(genePanelId);
+            const isInGenePanel = !wholeGenome && genePanelId && genePanelId in genePanelIdToEntrezGeneIds && genePanelIdToEntrezGeneIds[genePanelId].includes(entrezGeneId);
+
+            if (wholeGenome || isInGenePanel) {
```

Comment:
this can be simplified to
map[nextSampleId] = wholeGenome || isInGenePanel

Reply:
Yeah, you are right. Code updated.